tests/features/bootstrap/FeatureContext.php

```diff
@@ -1374,4 +1377,56 @@ public function fillAutoCompleteField(string $field, string $value): void {
     $element->postValue(['value' => [$value]]);
   }
 
+  /**
+   * Installs the testing module for scenarios tagged with @errorPage.
+   *
+   * @BeforeScenario @errorPage
+   */
+  public function installErrorPageTestingModule() {
+    $this->toggleErrorPageTestingModule('install');
+
+    // The test writes to the PHP error log because it's in its scope to test
+    // fatal errors. But the testing bots might reject tests that are not ending
+    // with a clear log. We mark in PHP error log the beginning and the end of
+    // this expected log entry so that we can clear it later.
+    error_log('BEFORE@errorPage');
+  }
+
+  /**
+   * Uninstalls the testing module for scenarios tagged with @errorPage.
+   *
+   * @AfterScenario @errorPage
+   */
+  public function uninstallErrorPageTestingModule(): void {
+    $this->toggleErrorPageTestingModule('uninstall');
+
+    // The test writes to the PHP error log because it's in its scope to test
+    // fatal errors. But the testing bots might reject tests that are not ending
+    // with a clear log. We mark in PHP error log the beginning and the end of
+    // this expected log entry so that we can clear it later.
+    error_log('AFTER@errorPage');
+
+    // Clear the log for fatal error produced by this test.
+    $error_log = ini_get('error_log');
```

Comment:
I am not sure about this. It seems strange to mangle with the server logs...

Reply:
Well, I see no other way to clear the errors logged by the test, which are are logged in the scope of the test. At the end of the test we should clear them, otherwise the Continuous PHP will output them at the end and is suggesting this is a failed test. I would happily accept any alternative way to achieve this.

Reply:
I improved this spart.